pkg/version/release.go

```diff
@@ -3,10 +3,10 @@ package version
 // This file was generated by release_generate.go; DO NOT EDIT.
 
 // Version is the version number in semver format X.Y.Z
-var Version = "0.2.0"
+var Version = "0.3.0"
 
 // PreReleaseID can be empty for releases, "rc.X" for release candidates and "dev" for snapshots
-var PreReleaseID = "dev"
+var PreReleaseID = ""
```

Comment:
@Himangini I changed`PreReleaseID` to `""` , is that correct? The comment made me think I should do that, but wanted to check :smile: